paddle/fluid/inference/analysis/node.cc

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/inference/analysis/node.h"
+#include "glog/logging.h"
+#include "paddle/fluid/platform/enforce.h"
+
+namespace paddle {
+namespace inference {
+namespace analysis {
+
+std::vector<Dot::Attr> Value::dot_attrs() const {
+  return std::vector<Dot::Attr>({Dot::Attr("style", "filled,rounded"),

```

Comment:
what are these "style" "shape" thing?

Reply:
These are [DOT](https://www.graphviz.org/doc/info/lang.html) language's attributes, to control the visualization for graph debug.

Here, different `Node` might have the different visual effect, for example, `Operator` and `Variable`  looks different.

Reply:
![image](https://user-images.githubusercontent.com/328693/39965961-0b74c58a-56d6-11e8-9333-abde92662ec6.png)

Reply:
I see.